src/tests/test_magic.py

```diff
@@ -368,14 +370,41 @@ def test_json_in_select(ip):
     assert result == [('{"greeting": "Farewell sweet {person}"}',)]
 
 
-def test_close_connection(ip):
+def test_closed_connections_are_no_longer_listed(ip):
     connections = runsql(ip, "%sql -l")
     connection_name = list(connections)[0]
     runsql(ip, f"%sql -x {connection_name}")
     connections_afterward = runsql(ip, "%sql -l")
     assert connection_name not in connections_afterward
 
 
+def test_close_connection(ip, tmp_empty):
+    # open two connections
+    ip.run_cell("%sql sqlite:///one.db")
+    ip.run_cell("%sql sqlite:///two.db")
+
+    # close them
+    ip.run_cell("%sql -x sqlite:///one.db")
+    ip.run_cell("%sql --close sqlite:///two.db")
+
+    assert "sqlite:///one.db" not in Connection.connections
+    assert "sqlite:///two.db" not in Connection.connections
+
+
+def test_column_names_visible(ip, tmp_empty):
+    res = ip.run_line_magic("sql", "SELECT * FROM empty_table")
+
+    assert "<th>column</th>" in res._repr_html_()
+    assert "<th>another</th>" in res._repr_html_()
+
+
+@pytest.mark.xfail(reason="known parse @ parser.py error")
+def test_sqlite_path_with_spaces(ip, tmp_empty):
```

Comment:
Do we also test for windows? The issue was opened about windows, usually paths there have different behaviors than mac/linux

Reply:
the OS is configured via github actions, not here. There's an open issue about running the tests on windows

Reply:
Ok, same, let's keep the issue open until we're certain it's done (I know Windows currently fails).